lte/gateway/python/magma/subscriberdb/protocols/s6a_proxy_servicer.py

```diff
@@ -46,7 +46,6 @@ def AuthenticationInformation(self, request, context):
         aia = s6a_proxy_pb2.AuthenticationInformationAnswer()
         try:
             plmn = request.visited_plmn
-
```

Comment:
Revert this

Reply:
Done

Reply:
Was this really resolved? The goal is to completely omit s6a_proxy_servicer.py from this PR. So it should not show up in this PR after this change is reverted.